docs/tutorials/1_finetune.md

```diff
@@ -13,40 +25,111 @@ To finetune on the custom datasets, the users need to modify four parts in the c
 
 Then the new config needs to modify the model according to the keypoint numbers of the new datasets. By only changing `out_channels` in the keypoint_head.
+For example, we have 133 keypoints for COCO-WholeBody, and we have 17 keypoints for COCO.
 
 ```python
+channel_cfg = dict(
+    num_output_channels=133,  # from 17 to 133
+    dataset_joints=133,  # from 17 to 133
+    dataset_channel=[
+        list(range(133)),  # from 17 to 133
+    ],
+    inference_channel=list(range(133)))  # from 17 to 133
+
 # model settings
 model = dict(
     type='TopDown',
-    pretrained=None,
-    backbone=dict(type='ResNet', depth=18),
+    pretrained='https://download.openmmlab.com/mmpose/'
+    'pretrain_models/hrnet_w48-8ef0771d.pth',
+    backbone=dict(
+        type='HRNet',
+        in_channels=3,
+        extra=dict(
+            stage1=dict(
+                num_modules=1,
+                num_branches=1,
+                block='BOTTLENECK',
+                num_blocks=(4, ),
+                num_channels=(64, )),
+            stage2=dict(
+                num_modules=1,
+                num_branches=2,
+                block='BASIC',
+                num_blocks=(4, 4),
+                num_channels=(48, 96)),
+            stage3=dict(
+                num_modules=4,
+                num_branches=3,
+                block='BASIC',
+                num_blocks=(4, 4, 4),
+                num_channels=(48, 96, 192)),
+            stage4=dict(
+                num_modules=3,
+                num_branches=4,
+                block='BASIC',
+                num_blocks=(4, 4, 4, 4),
+                num_channels=(48, 96, 192, 384))),
+    ),
     keypoint_head=dict(
         type='TopdownHeatmapSimpleHead',
-        in_channels=512,
-        out_channels=17,
-    ),
+        in_channels=48,
+        out_channels=channel_cfg['num_output_channels'], # modify this
+        num_deconv_layers=0,
+        extra=dict(final_conv_kernel=1, ),
+        loss_keypoint=dict(type='JointsMSELoss', use_target_weight=True)),
     train_cfg=dict(),
     test_cfg=dict(
         flip_test=True,
         post_process='unbiased',
         shift_heatmap=True,
-        modulate_kernel=11),
-    loss_keypoint=dict(type='JointsMSELoss', use_target_weight=False))
+        modulate_kernel=17))
 ```
 
+Note that the `pretrained='https://download.openmmlab.com/mmpose/pretrain_models/hrnet_w48-8ef0771d.pth'` setting is used for initializing backbone.
+If you are training a new model from ImageNet-pretrained weights, this is for you.
+However, this setting is not related to our task at hand. What we need is load_from, which will be discussed later.
+
 ## Modify dataset
 
-The users may also need to prepare the dataset and write the configs about dataset. MMPose already support COCO and MPII-TRB Dataset.
+The users may also need to prepare the dataset and write the configs about dataset.
+MMPose already support multiple (10+) dataset, including COCO, COCO-WholeBody and MPII-TRB datasets.
```

Comment:
support -> supports. And maybe removing the word datasets at the end?

Reply:
~~already~~